cmd/clusterctl/cmd/version_checker.go

```diff
@@ -91,6 +91,13 @@ type VersionState struct {
 // version is the same or greater it returns nothing.
 func (v *versionChecker) Check() (string, error) {
 	log := logf.Log
+
+	if versionFileLink, _ := os.Readlink(v.versionFilePath); versionFileLink == "/dev/null" {
```

Comment:
This seems very untransparent as a way to handle this. Is there a reason to not have this as a flag?